src/Files.App/Program.cs

```diff
@@ -13,20 +12,31 @@
 
 namespace Files.App
 {
+	/// <summary>
+	/// Represents the root entry point for the Files app assembly.
+	/// </summary>
 	internal class Program
 	{
-		// Note:
-		//  We can't declare Main to be async because in a WinUI app
-		//  This prevents Narrator from reading XAML elements
-		//  https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/CsWinUiDesktopInstancing/Program.cs
-		//  STAThread has no effect if main is async, needed for Clipboard
+		private const uint CWMO_DEFAULT = 0;
+
+		private const uint INFINITE = 0xFFFFFFFF;
```

Comment:
Where are these coming from?

Reply:
Coming from 153-155. Just moved to the top according to the guideline.